src/index.html

```diff
@@ -5,4 +5,151 @@
+        <p>
+            REVIEW CALENDAR, TUITION, REGISTRATION PROCEDURES<br>
+            Double ​check the classes you would like to take. Some classes meet twice per week. Make note of any potential conflicts and talk to us if you have one.
+            Commit to the year or join us for our 13-Week Session! Save money by paying in full. You will otherwise be put on a monthly payment plan.</p>
+    </div>
+
+    <button class="accordion">Step 3</button>
+    <div class="panel">
+        <p>
+            REGISTER ONLINE<br>
+            Log in to your account, or create a new one!
+            Once you've selected your classes, we will review and approve your registration.​ We automatically charge a deposit for registration fee, September tuition, and costume deposit(s) to secure your space in the classes you've selected. You will then be sent a confirmation email and balance due for the season.</p>
+    </div>
+    <button class="accordion">Step 4</button>
+    <div class="panel">
+        <p>
+            ORDER YOUR UNIFORMS AND MSDC GEAR<br>
+             We have created a convenient list of approved, affordable attire for classes through Curtain Call for Class​. You can also order MSDC gear through this link! All students are required to be in uniform; Repertory students must also purchase a sweatsuit.</p>
+    </div>
+    <button class="shedule">Schedule</button>
+    <button class="tuition">Tuition</button>
+</div>
+<div class="contact">
+    <h1>Get In Touch</h1>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3086.1248737884016!2d-76.6446764842207!3d39.330756228887225!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c81b2abe742d9b%3A0xa8f579926b1381e!2sMorton%20Street%20Dance%20Center!5e0!3m2!1sen!2sus!4v1605202579180!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
+    <form>
+        <label for="fname">First name:</label><br>
+        <input type="text" id="fname" name="fname"><br>
+
+        <label for="lname">Last name:</label><br>
+        <input type="text" id="lname" name="lname"><br>
+
+        <label for="email">Email:</label><br>
+        <input type="text" id="email" name="email"><br>
+
+        <label for="phone">Phone:</label><br>
+        <input type="text" id="phone" name="phone"><br>
+
+        <label for="message">Type your message here...</label><br>
+        <input type="text" id="message" name="message"><br>
+
+        <input type="submit" value="Submit">
+      </form>
+    <button class="tour">Request a Tour</button>
+    <button class="app">Download Our App</button>
+    <button class="subscribe">Subscribe</button>
+</div>
+
+<div class="footer">
+    <h1>Morton Street Dance Center</h1>
+    <ui class="sitemap">
+        <li class="title">Site Map</li>
+        <li>About</li>
+        <li>Classes</li>
+        <li>Registration</li>
+        <li>Contact</li>
+    </ui>
+    <ui class="smedia">
```

Comment:
I think you mean to use the `ul` element here and on the closing element for this list.